### ISSUE TITLE
Not handling top level functions correctly

### ISSUE THREAD

**Issue by pogson** [readthedocs/sphinx-autoapi#177]
**Describe the bug**
When creating documentation for a module with top level functions the generated rst file is incorrect. The generated rst is expected to have a blank line after `... function::`

**To Reproduce**
Create a module with
```
def strange():
    """
    This should be a description
    """
    
    pass

def expected():
    """
    :noindex:

    This is a description
    """

    pass
```

Here's generated rst file
```
:mod:`example`
==============

.. function:: strange()
   This should be a description


.. function:: expected()
   :noindex:

   This is a description
```

Here's expected rst file. note the space after the `... function::` line
```
:mod:`example`
==============

.. function:: strange()

   This should be a description


.. function:: expected()
   :noindex:

   This is a description
```

**Screenshots and Expected behavior**
Classic Theme:
![example-classic](https://user-images.githubusercontent.com/44173402/65979670-4b5b7880-e43b-11e9-9793-d57935d558ce.PNG)

Sphinx-rtd:
![example-sphinx-rtd](https://user-images.githubusercontent.com/44173402/65979671-4b5b7880-e43b-11e9-94d6-11d284f6155c.PNG)

**Environment info**
- OS: Ubuntu 18.04.1 LTS
- Python version: 3.6.8
- Sphinx version: 2.2.0
- Sphinx extensions: None

**Comment by AWhetter** [comment]
Which version of sphinx-autoapi are you using? This should have been fixed since v1.1.0 (https://github.com/readthedocs/sphinx-autoapi/commit/9d45eed4d0c44031fdbe81b7dcf660d2dbfe82ef).

**Comment by pogson** [comment]
Ah, you're right. I just installed it. I wonder why I got version 1.0.0. This appears to be working.
